model-optimizer/extensions/back/MarkNodesWithShapeValues.py

```diff
@@ -2,12 +2,13 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import logging as log
+from typing import Callable, List, Dict
 
 import numpy as np
 
 from extensions.middle.MarkSubgraphsWithCorrectLayout import MarkSubGraphsWithCorrectLayout
 from mo.back.replacement import BackReplacementPattern
-from mo.graph.graph import Graph
+from mo.graph.graph import Graph, Node
```

Comment:
please sort imports. Use `Ctrl+Alt+O` and `Ctrl+Alt+A` for code normalization in PyCharm